Altis remastered

wersal454 opened the pull request:
## What type of PR is this?
1. [ ] Bug
2. [x] Change
3. [ ] Enhancement
4. [ ] Miscellaneous

### What have you changed and why?
Information:

As the name suggests, remastering of Altis map

Added more locations, some inspired by campaign and dlc's, some I got out of my head

Moved and changed exiting locations

Might have overkilled it

### Please specify which Issue this PR Resolves (If Applicable).
"This PR closes #XXXX!"

### Please verify the following.

1. [x] Have you loaded the mission in LAN host?
2. [ ] Have you loaded the mission on a dedicated server?

### Is further testing or are further changes required?

1. [ ] No
2. [x] Yes (Please provide further detail below.)

### How can the changes be tested?
Steps:

********************************************************
Notes:

wersal454 committed:
Update mission.sqm

wersal454 committed:
Altis remastered

wersal454 commented:
Would love to see ideas and suggestions

Because this is a first time I ever done Antistasi map, I probably messed up somewhere


I didn't update navgrid, I think I could do it after general review

Also I didn't lauch TownGenerator, I don't think it's needed, though might need to double check, just in case

wersal454 commented:
John from community told me that, function to populate marker areas is actually taking the largest value of markerSize x & y, and uses that for an initial radius check before checking against the marker.

So if you place towers or other buildings in corners, it will lose them and won't spawn troops in them. Even though you set your marker to be a square

So remember kids, all squares are actually circles

jwoodruff40 commented:
> John from community told me that, function to populate marker areas is actually taking the largest value of markerSize x & y, and uses that for an initial radius check before checking against the marker.
> 
> So if you place towers or other buildings in corners, it will lose them and won't spawn troops in them. Even though you set your marker to be a square
> 
> So remember kids, all squares are actually circles

So maybe instead of using `selectMax markerSize _markerX` we should use

```
private _markerW = (markerSize _markerX) select 0;
private _markerH = (markerSize _markerX) select 1;
private _markerRadius = (_markerW^2 + _markerH^2)^0.5;
```

(I'm just guessing on the code implementation here  I don't know where specifically John is referring to)

wersal454 committed:
better way to determinate marker size

wersal454 committed:
syntax

syntax

wersal454 commented:
Thanks to @jwoodruff40, a better way to determent square marker size

hopefully it won't f everything up

wersal454 commented:
after thinking about it a bit more, I think of least 1 major issue with new method

It can cause area overlapping which will spawn statics inside towers twice (for example)

So, I guess, better way needs to be developed, which to my understanding is in the works already

wersal454 commented:
> after thinking about it a bit more, I think of least 1 major issue with new method
> 
> It can cause area overlapping which will spawn statics inside towers twice (for example)
> 
> So, I guess, better way needs to be developed, which to my understanding is in the works already

Disregard, I've tested it and apparently there is an additional check to prevent double spawn, so all good.

wersal454 committed:
very simple object and draw distance settings

Added as separate CBA settings since antsistasi somehow blocks LOD settings

jwoodruff40 commented:
Would probably make sense to make the last few commits their own PR and then just have this Altis change PR be dependent on that one.

stutpip123 commented:
you know the gamemode completely bricks if you have too many resource zones...
needless to say the current layout of about 30-50 of them is too much, especially in clusters of 3-4 in multiple places

stutpip123 commented:
some areas around the top right are currently still set to occupants when i think they should be invader

stutpip123 commented:
a lot of the added areas are very... empty. with usually nothing more than a cargo building or two slapped down. if you are gonna add areas, better to add just a few with more interesting layouts than to just spam stuff anywhere that it might fit

wersal454 commented:
> you know the gamemode completely bricks if you have too many resource zones... needless to say the current layout of about 30-50 of them is too much, especially in clusters of 3-4 in multiple places

welp, didn't know that

wersal454 commented:
> some areas around the top right are currently still set to occupants when i think they should be invader

I think Seaport, resource point and outpost near top right city are fine to give to Occupants

No point in giving Invaders the outpost since they don't take over cities, so it's kind of city in encirclement situation

Seaport and resource point is kind of the last life line for that city

Idk seems more interesting this way

wersal454 commented:
> a lot of the added areas are very... empty. with usually nothing more than a cargo building or two slapped down. if you are gonna add areas, better to add just a few with more interesting layouts than to just spam stuff anywhere that it might fit

Not sure what exactly are you talking about, but "empty" areas were a thing before, some don't even have cargo buildings

Areas that I've added, I've tried giving them more an outpost vibe

resource points are indeed empty, because there were empty before (and kinda always are iIrc)

So if you have any suggestions, send some screenshots

wersal454 committed:
resource areas reduction

stutpip123 commented:
![image](https://github.com/user-attachments/assets/67a09120-500e-42e2-99c1-229b7bdb458f)
Tanoa?

wersal454 commented:
> ![image](https://private-user-images.githubusercontent.com/77023662/429162963-67a09120-500e-42e2-99c1-229b7bdb458f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDM1MzAwODcsIm5iZiI6MTc0MzUyOTc4NywicGF0aCI6Ii83NzAyMzY2Mi80MjkxNjI5NjMtNjdhMDkxMjAtNTAwZS00MmUyLTk5YzEtMjI5YjdiZGI0NThmLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTA0MDElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwNDAxVDE3NDk0N1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTkwYTc1MDU4ZjIxOTVhNjczYjgxN2Q2YzZjNjRjODcwODg1OTI5ZTE5ZDJmMGIzYjc3ZjQ4ZmM2ZjU0Y2U1ZmQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.dij4DQPMakXKxwxLLfj8CmCT1TBWpwQjvhncQgoE6_M) Tanoa?

![image](https://github.com/user-attachments/assets/5c037c0d-cf3a-4fa7-9c9e-999365e0cb94)

wersal454 committed:
resource areas reduction

khm

wersal454 committed:
mortars names fix

wersal454 committed:
helipads over objects support

Option to create helicopter spawn positions over/above objects by using userTexture object

wersal454 committed:
small adjustments for objects spawning

mg will spawn a bit closer to the windows on cargo Patrol towers
(hopefully) fixed exploding parked civilian vehicles due to them colliding with walls and such
made it so every radio antennas are always straight up

wersal454 committed:
populate outpost hangars

wersal454 committed:
map adjustments

wersal454 committed:
Update fn_createAmbientCivTraffic.sqf

wersal454 committed:
small navgrid adjustments

wersal454 committed:
Merge branch 'unstable' into Altis-remastered

Signed-off-by: wersal <wroma@bk.ru>

wersal454 committed:
Merge branch 'unstable' into Altis-remastered

wersal454 committed:
quick fix

wersal454 committed:
Update fn_settings.sqf

wersal454 committed:
Merge branch 'unstable' into Altis-remastered

Signed-off-by: wersal <wroma@bk.ru>

wersal454 committed:
Update CfgVehicles.hpp

wersal454 committed:
Merge branch 'unstable' into Altis-remastered

Signed-off-by: wersal <wroma@bk.ru>
